packages/bpk-component-checkbox/src/BpkCheckbox.js

```diff
@@ -88,7 +97,7 @@ const BpkCheckbox = (props: Props) => {
         className={inputClasses}
         name={name}
         disabled={disabled}
-        aria-label={label}
+        aria-label={ariaLabelToUse()}
```

Comment:
You could use a ternary here `aria-label={ariaLabel ? ariaLabel : label}

Reply:
way nicer solution, thanks